Sources/ArcGISToolkit/Resources/ar.lproj/Localizable.strings

```diff
@@ -1,2 +1,281 @@
+/* A label declaring the number of starting points selected for a utility network trace. */
+"%lld selected" = "بيت_%lld selected______________لاحقة";
```

Comment:
It seems that a majority, if not all, of the translations have these underscores. I double checked some of our older translated projects and didn't see the same. Are they expected?

Reply:
@dfeinzimer this is first time Pseudo and is expected for pre translation work.